include/odp/api/spec/timer.h

```diff
@@ -37,15 +37,32 @@ extern "C" {
 
 /**
  * Clock sources for timers in timer pool.
+ *
+ * ODP_CLOCK_CPU is the default clock source and it is supported always. It is implementation
+ * defined which other clock sources are supported. See implementation documentation for
```

Comment:
The notion of external clock, probably, needs to be defined more explicitly. For example the clock may come from the GPIO pin or some SOC internal source (PLL). Are they both external in this case?

Reply:
Basically, ODP_CLOCK_CPU is the default (may be internal or external), EXT_0 ... 3  are implementation defined where those connect.

We could also use term ODP_CLOCK_SRC_0 ... _3, and leave EXT if it's confusing. Timer API does not have many references to EXT, so the naming could be changed.